### PR TITLE
Fix coin

### PR DESCRIPTION
**Pull request by SkyRiS3s**

### CODE REVIEW

File: app/src/main/java/ch/epfl/sdp/game/StartGameController.java [hunk 1]
```diff
@@ -79,9 +79,9 @@ Area initGameArea() {
         //GameArea -----------------------------------------
         GeoPoint local = PlayerManager.getInstance().getCurrentUser().getLocation();
         Area gameArea = new CircleArea(3000, local);
-        gameInstance.addToDisplayList(gameArea);
-        gameInstance.addToUpdateList(gameArea);
-        gameInstance.areaShrinker.setGameArea(gameArea);
+         Game.getInstance().addToDisplayList(gameArea);
+         Game.getInstance().addToUpdateList(gameArea);
+         Game.getInstance().areaShrinker.setGameArea(gameArea);
```
Comment (RaphaelBonatti) [inline app/src/main/java/ch/epfl/sdp/game/StartGameController.java:84]: Why did you change that?

Comment (Zeavee) [reply app/src/main/java/ch/epfl/sdp/game/StartGameController.java:84]: Surely merging issues
